src/doc/rustc/src/platform-support/riscv32-unknown-none-elf.md

```diff
@@ -1,12 +1,12 @@
-# `riscv32{i,im,ima,imc,imac,imafc}-unknown-none-elf`
+# `riscv32{i,im,ima,imc,imac,imafc,e,em,emc}-unknown-none-elf`
 
 **Tier: 2**
 
 Bare-metal target for RISC-V CPUs with the RV32I, RV32IM, RV32IMC, RV32IMAFC and RV32IMAC ISAs.
 
 **Tier: 3**
 
-Bare-metal target for RISC-V CPUs with the RV32IMA ISA.
+Bare-metal target for RISC-V CPUs with the RV32IMA, RV32E, RV32EM and RV32EMC ISAs.
```

Comment:
Or we can add documentation that we don't actually generate code with a 4-byte stack alignment, but rather to a 16-byte stack alignment, which is I guess technically acceptable even if it is incorrect.